test/mbr_update_test.rb

```diff
@@ -28,6 +31,21 @@
       )
     end
 
+    context "on s390" do
+      before do
+        allow(Yast::Arch).to receive(:architecture).and_return("s390_64")
```

Comment:
Actually here we're depending on an implementation detail of the `Arch` class. Instead:
```ruby
allow(Yast::Arch).to receive(:s390).and_return(true)
```

and `and_return(false)` for the default case. Then the default case is depending on *our* implementation details which is better.